merlin/models/utils/data_etl_utils.py

```diff
@@ -19,6 +20,23 @@
 logger.setLevel(logging.INFO)
 
 
+def get_movielens(path=None, variant="ml-25m"):
+    """Returns a tuple of train/test Dataset objects for the movielens dataset"""
```

Comment:
Could use docs for the parameters (mostly to show the options for variant). I’m wondering if it would be better to rename `variant` to `size` and allow for `{100k, 25m}`

Reply:
I've added a better docstring in the last commit - 

I left the parameter name as variant, size there are dataset differences beyond just the number of rows (like the ml-100k dataset has extra user metadata that the ml-25m dataset doesn't have). I don't have a strong opinion on this though